### ISSUE TITLE
Cannot bridge view once media

### ISSUE THREAD

**Issue by hoangvietfreelancer** [mautrix/meta#106]
I got this error in matrix when testing with view once both video and image
```
Unsupported message (*waArmadilloApplication.Armadillo_Content_RavenMessageMsgr)

Please open in Messenger app
```
In the bridge logs:
```log
2024-10-14T10:11:28.729Z WRN Unrecognized armadillo content type action="handle remote event" bridge_evt_type=RemoteEventMessage content_type=*waArmadilloApplication.Armadillo_Content_RavenMessageMsgr event_loop_index=21 message_id=7251535068118852856 portal_id=8341548332618616 portal_mxid=!ZwdPctYFgwdzPMpVng:example.com sender_id=100045218265910:4@msgr source_id=100045218265910
```

**Comment by TristanL06** [comment]
View once message seems to not being displayed as "unsupported message" most of the time. It is just not displayed at all.
Once I had the "unsupported message, Open in Instagram : <link>" 

android version: 14
beeper version: 4.19.0

**Comment by ireun** [comment]
I don't think it can be fixed - these types of media cannot be seen even on official web apps

**Comment by TristanL06** [comment]
It should at least be marked as "not supported image" to inform that there is a content, as it is in the officiel webapp. For now we don't even know when we receive one of this images.

**Comment by highda** [comment]
Chiming in to propose the `direct_v2/threads/{thread_id}/` endpoint for mobile devices, which can actually retrieve this type of message along with the media URL. Just my 2 cents. Might mess around and fork, but I'd have to take a GoLang 101 crash course first 😅

**Comment by devnoname120** [comment]
I think the https://github.com/mautrix/instagram bridge used to support `view-once` media? Not 100% sure

**Comment by leo-grnd** [comment]
This looks like it was fixed a while ago. The unhandled type in the report,
`Armadillo_Content_RavenMessageMsgr`, now has a case in the armadillo content
switch in `pkg/msgconv/from-whatsapp.go`, added in a6aef6a
("msgconv/from-whatsapp: add support for raven messages", 2026-02-19).

Probably safe to close.
